_posts/convars/2020-10-25-mom_gamemode_override.md

```diff
@@ -0,0 +1,28 @@
+---
+title: mom_gamemode_override
+category: var
+tags:
+  - gamemode
+minimum_value: 0
+maximum_value: Same as gamemode count
```

Comment:
Since we're pretty set on gamemodes now and to bring in line with other cvars, I'd put the actual value in.
```suggestion
maximum_value: 10
```